docs/development/decision-records/2024-09-16-proposal-federated-catalog-distribution/README.md

```diff
@@ -25,7 +27,12 @@ Since the Federated Catalog will be a standalone runtime, the Tractus-Connector
 To enable the Federated Catalog flow, please [see this table](https://github.com/eclipse-tractusx/tractusx-edc/blob/75bdacbad43e2cad352204ea28a359c6aac7adea/docs/development/management-domains/README.md#enable-and-configure-the-crawler-subsystem).
 
 For its TargetNodeDirectory, the user is able to obtain the Connectors' URL's through the Discovery Service and store them in the new extension through its API. The API will allow to save a list of BPNLs (and Connectors' URL's if desired) and the `DiscoveryServiceRetrieverExtension` is responsible to retrieve the data and store it (in memory or in a database). The URL's can later be retrieved and crawled by the Federated Catalog.
-This solution improves on the default one of having the data in a static file since a dynamic approach would avoid downtime when a change is required.
+
+The retrieval of Connector URL's through the Discovery Service is enabled by the endpoint:
+```
+POST: /api/administration/connectors/discovery
```

Comment:
Is there any api spec document from the Discovery Service documentation that can be referenced here?

Reply:
I did not find much, been using a int environment swagger to test it, however regarding DS guideline usage it can be found https://catenax-ev.github.io/docs/standards/CX-0001-EDCDiscoveryAPI#22-api-specification. Added in the PR [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/1555/commits/b4b00b005dcbc5943738eeb04f4dfd9ec40722aa).

Reply:
We need to discuss this. The Discovery Service will likely be retired in 2025. We may need to use it as a temporary solution, but then this API should remain alpha until then.

Reply:
We were not aware of the 2025 timeline for the DiscoveryService. We had a proposal that would not enforce the DS querying in this extension by having the member responsible to obtain the Participant data, but since that would create more manual work, the proposal was then to query the DS (since we had no idea of it ceasing to be used).